lib/testers/rule-tester.js

```diff
@@ -69,8 +69,8 @@ const RuleTesterParameters = [
     "code",
     "filename",
     "options",
-    "args",
-    "errors"
+    "errors",
+    "output"
```

Comment:
Can we add a test to `rule-tester` to verify that an error is thrown when an unknown property is included in a test?